run-build-functions.sh

```diff
@@ -102,6 +103,32 @@ run_yarn() {
   fi
 
 
+  if [ "$NETLIFY_YARN_WORKSPACES" = "true" ]
+  then
+    echo "NETLIFY_YARN_WORKSPACES feature flag set"
+    # Ignore path env var required in order to support yarn 2 projects
+    if YARN_IGNORE_PATH=1 yarn workspaces info
```

Comment:
[sand] Can we clarify a bit what `YARN_IGNORE_PATH` does?

Reply:
Tried to had a better description. Let me know what you think @erezrokah 👍 The naming for the variable vs its final purpose can be a bit confusing 😅

Reply:
@JGAntunes Are these env vars intended to be user-editable in the long term? If so, we'll need to document them. (cc @rstavchansky)

Is there anything else that should be documented? Or is this automatic behavior that's considered more "under the hood"?

Reply:
Thanks @JGAntunes - so we are forcing `yarn@1` just for the workspaces command? 
And this works since the configuration hasn't changed between v1 and v2?

Reply:
> Are these env vars intended to be user-editable in the long term?
> Is there anything else that should be documented? Or is this automatic behavior that's considered more "under the hood"?

The only new user-editable env var we're introducing is `NETLIFY_YARN_WORKSPACES`, and it shouldn't be editable in the long term, hence no need to document it IMO CC @rstavchansky. Theoretically speaking, users could set an environment variable `NETLIFY_YARN_WORKSPACES=true` that would trigger this behaviour (instead of relying on a feature flag on our side to do it - which is our plan to roll this out) but I don't really see that as major concern. The plan is for this behaviour to eventually become default without the need of any env vars (we infer the presence of a `workspace` monorepo on our end).

That being said, later on (maybe once this has been rolled out accordingly), I think it would be a good idea to produce some examples of monorepo usages with Netlify as it is not a trivial setup to do and, to my understanding, we don't have a write up on this (correct me if I'm wrong though). I think there's a lot of workarounds that originated in the issues above that are no longer applicable, so documenting that could probably be beneficial. Happy to move this discussion back to - https://github.com/netlify/pod-workflow/issues/139 - if you prefer 👍 CC @rstavchansky @verythorough

Reply:
> Thanks @JGAntunes - so we are forcing yarn@1 just for the workspaces command?
And this works since the configuration hasn't changed between v1 and v2?

Exactly @erezrokah! The configuration is still extracted from the `workspaces` entry in the `package.json` and that hasn't changed (from what I managed to look into), that way we are able to validate the presence of `workspaces` without the need of a separate branch for the `yarn berry` case.

Is that clear enough from the comment and usage? I'm happy to provide further information in there if you think like it makes sense 👍